AutoDefineAddon/autodefine.py

```diff
@@ -285,8 +284,11 @@ def _get_definition(editor,
                 final_pronounce_index = fields.index(field)
                 break
 
-        for sound_local_filename in reversed(all_sounds):
-            insert_into_field(editor, '[sound:' + sound_local_filename + ']', final_pronounce_index)
+        to_print = ""
+        for sound_local_filename in all_sounds:
+            to_print += f'[sound:{sound_local_filename}]'
+        to_print += "<br>"
```

Comment:
I'd like to avoid newlines at the end of the `[sound:]` field. Perhaps for the phonetic transcription, we can check to see if the field is non-empty and add a newline at that point?

Reply:
I applied your point and will open another pull request for improving the arrangement.